core/src/test/java/io/grpc/internal/ManagedChannelImplTest.java

```diff
@@ -3376,6 +3381,59 @@ public void getAuthorityAfterShutdown() throws Exception {
     assertEquals(SERVICE_NAME, channel.authority());
   }
 
+  @Test
+  public void nameResolverHelper_emptyConfigSucceeds() {
+    int defaultPort = 1;
+    ProxyDetector proxyDetector =  GrpcUtil.getDefaultProxyDetector();
+    SynchronizationContext syncCtx =
+        new SynchronizationContext(Thread.currentThread().getUncaughtExceptionHandler());
+    boolean retryEnabled = false;
+    int maxRetryAttemptsLimit = 2;
+    int maxHedgedAttemptsLimit = 3;
+
+    NrHelper nrh = new NrHelper(
+        defaultPort,
+        proxyDetector,
+        syncCtx,
+        retryEnabled,
+        maxRetryAttemptsLimit,
+        maxHedgedAttemptsLimit);
+
+    ConfigOrError<ManagedChannelServiceConfig> coe =
+        nrh.parseServiceConfig(ImmutableMap.<String, Object>of());
+
+    assertThat(coe.getConfig()).isNotNull();
```

Comment:
Should be useful to have `(coe.getError()).isNull()`.

Reply:
This is impossible, because of ConfigOrError.  There can't be both.